### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Tatitooo**

### CODE REVIEW

File: source/BP-Essentials/Chat/Commands/Everyone/GetKit.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public static void Run(SvPlayer player, string message)
                 player.SendChatMessage($"<color={errorColor}>That kit is currently disabled.</color>");
                 return;
             }
-            if (obj.CurrentlyInCooldown.ContainsKey(player.player.username))
+            if (obj.CurrentlyInCooldown.ContainsKey(player.player.username && obj.CurrentlyInCooldown[player.player.username] > 0)
```
Comment (PLASMAchicken) [inline source/BP-Essentials/Chat/Commands/Everyone/GetKit.cs]:
```suggestion
            if (obj.CurrentlyInCooldown.ContainsKey(player.player.username) && obj.CurrentlyInCooldown[player.player.username] > 0)
```

File: source/BP-Essentials/HookMethods.cs [hunk 2]
```diff
@@ -31,6 +31,12 @@ public static void Initialize(SvEntity player)
             if (svPlayer == null)
                 return;
             PlayerList.Add(svPlayer.player.ID, new PlayerListItem(svPlayer.player));
+        }
+}
```
Comment (PLASMAchicken) [inline source/BP-Essentials/HookMethods.cs]:
```suggestion
```

File: source/BP-Essentials/Methods/Check/CheckBanned.cs [hunk 3]
```diff
@@ -18,14 +18,20 @@ public static void Run(SvPlayer player)
...
                         return;
+                    }
                     foreach (var line in File.ReadAllLines(BansFile))
-                        if (string.IsNullOrEmpty(line) || !line.StartsWith("# " + player.player.username, StringComparison.CurrentCulture))
-                            continue;
-                    Debug.Log($"{PlaceholderParser.ParseTimeStamp()} [WARNING] {player.player.username} Joined while banned! IP: {player.connection.IP}");
-                    player.svManager.AddBanned(player.player);
-                    player.svManager.Disconnect(player.connection, DisconnectTypes.Banned);
+                    {
+                        if (line.Trim() == $"# {player.player.username}".Trim()) || line == player.connection.IP.Trim())
```
Comment (PLASMAchicken) [inline source/BP-Essentials/Methods/Check/CheckBanned.cs]:
```suggestion
                        if (line.Trim() == $"# {player.player.username}".Trim() || line == player.connection.IP.Trim())
```

Comment (Tatitooo) [reply source/BP-Essentials/Methods/Check/CheckBanned.cs]: Yeah, i just did that without opening visual studio, i know i should post that, just were fixing and didn't check if there was a problem i had to go and i posted that, thanks you ;)

**Pull request closed without merging**